core/src/main/java/org/kiwix/kiwixmobile/core/main/KiwixWebView.kt

```diff
@@ -57,7 +57,7 @@ open class KiwixWebView @SuppressLint("SetJavaScriptEnabled") constructor(
   webViewClient: CoreWebViewClient
 ) : VideoEnabledWebView(context, attrs) {
   @Inject
-  lateinit var sharedPreferenceUtil: SharedPreferenceUtil
+  open lateinit var sharedPreferenceUtil: SharedPreferenceUtil
```

Comment:
not a fan of this, You override this field in the constructor, so it gets passed in externally and then I really don't know what happens when inject is called but it might reassign the variable... to the same object. It is a lot of steps. We can just pass sharedPrefs to the constructor of KiwixWebView and get rid of the injection here